charts/sysdig-deploy/tests/notes_test.yaml

```diff
@@ -72,3 +72,37 @@ tests:
     asserts:
       - failedTemplate:
           errorMessage: "raw: global.sysdig.region=ap3 provided is not recognized."
+
+  - it: Test agent.monitor.enabled when specifying Agent secure_light mode
+    set:
+      agent:
+        monitor:
+          enabled: true
```

Comment:
If we are testing object creation, we must also test the other way round (test that no objects are created if `false'). 🙃